README.md

```diff
@@ -1,58 +1,110 @@
 
-This script mirrors automatically the public repositories from a github-user or github-organization to your gitea server.
-It will - once started - create a mirrored repository under a given token for a gitea user fully automatically.
+This script automatically mirrors the repositories from a github-user or github-organization to your gitea server.
+Once started, it will create a mirrored repository under a given token for a gitea user, completely automatically.
 
 Example:
 A github user `github-user` has public repositories `dotfiles` and `zsh-config`.
-Starting the script with a gitea token for the account `gitea-user` will create the following mirror repositories:
+Starting the script with a gitea token for the account `gitea-user` will create the following mirrored repositories:
 
-- github.com/github-user/dotfiles &larr; some-gitea.url/gitea-user/dotfiles
-- github.com/github-user/zsh-config &larr; some-gitea.url/zsh-config/dotfiles
+- github.com/github-user/dotfiles &rarr; your-gitea.url/gitea-user/dotfiles
+- github.com/github-user/zsh-config &rarr; your-gitea.url/gitea-user/zsh-config
 
 The mirror settings are default by your gitea instance.
 
+It is also possible to mirror private repos, which can be configred here in [#parameters](#parameters). When mirroring private repos, they will be created as private repos on your gitea server.
+
 ## Prerequisites
 
-- Something to mirror (a github user or organization with public repos)
-- Gitea instance up and running
-- User for Gitea with generated token
-- Docker
+- A github user or organization with repositories
+- Configured Gitea instance up and running
+- User for Gitea with generated token (Settings -> Applications -> Generate New Token)
+- Docker or Docker Compose
 
-## Run it
+### Docker Run
 
 ```sh
-docker container run \
+docker run \
  -d \
  --restart always \
  -e GITHUB_USERNAME=github-user \
- -e GITEA_URL=https://some-gitea.url \
+ -e GITEA_URL=https://your-gitea.url \
  -e GITEA_TOKEN=please-exchange-with-token \
  jaedle/mirror-to-gitea:latest
 ```
 
-This will a spin up a docker container running infinite which will try to mirror all your repositories once every hour to your gitea server.
+This will a spin up a docker container which will run forever, mirroring all your repositories once every hour to your gitea server.
+
+### Docker Compose
+
+```yaml
+version: "3.3"
+services:
+    mirror-to-gitea:
+        image: jaedle/mirror-to-gitea:latest
+        restart: always
+        environment:
+          - GITHUB_USERNAME=github-user
+          - GITEA_URL=https://your-gitea.url
+          - GITEA_TOKEN=please-exchange-with-token
+          #- GITHUB_TOKEN=please-exchange-with-token # Optional, set to mirror private repos
+          #- MIRROR_PRIVATE_REPOSITORIES=true # Optional, set to mirror private repos
+          # - DELAY=3600 # Optional, set to change the delay between checks (in seconds)
+        container_name: mirror-to-gitea
+```
+## Building from Source
```

Comment:
I think building from source is valuable so that people can see what the docker container is running when it is spun up, but `./run-local.sh` is definitely helpful for testing too. When I was testing I made sure it ran as a script first, then would use compose to spin up a container for further testing.